spec/unit/installer_spec.rb

```diff
@@ -351,6 +351,59 @@ def @installer.clean_pod_sources
         UI.warnings.should.include 'BlocksKit has been deprecated'
       end
 
+      it 'prints a warning if the master specs repo is not explicitly used but it exists in the users repos dir' do
+        podfile = Pod::Podfile.new do
+          install! 'cocoapods', :integrate_targets => false
+          platform :ios
+        end
+        @installer = Installer.new(config.sandbox, podfile)
+        master_source = Source.new(Pod::Installer::MASTER_SPECS_REPO_GIT_URL)
+        master_source.stubs(:url).returns(Pod::Installer::MASTER_SPECS_REPO_GIT_URL)
+        config.sources_manager.stubs(:all).returns([master_source])
+        @installer.send(:warn_for_removing_git_master_specs_repo)
+        UI.warnings.should.include 'Your project does not explicitly specify the CocoaPods master specs repo. Since CDN is now used as the' \
+          ' default, you may safely remove it from your repos directory via `pod repo remove master`.' \
```

Comment:
@igor-makarov is `pod repo remove master` correct to delete the old master specs repo?  We do not do any migration correct? 

Basically we keep `master` and add `trunk` for users who upgrade from an older version of CP to one that uses CDN.

Reply:
looks right to me